Formula/l/llama-cpp.rb

```diff
@@ -0,0 +1,34 @@
+class LlamaCpp < Formula
+  desc "LLM inference in C/C++"
+  homepage "https://github.com/ggerganov/llama.cpp"
+  url "https://github.com/ggerganov/llama.cpp/archive/refs/tags/b2780.tar.gz"
+  sha256 "8b1831b605ee4c564ff84e648dabdc11fe3d77dcb6b1d7e4d97a14ea991755f2"
+  license "MIT"
+  head "https://github.com/ggerganov/llama.cpp.git", branch: "master"
+
+  livecheck do
+    throttle 10
+  end
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+
+    libexec.install "build/bin/default.metallib" if OS.mac?
+    libexec.install "build/bin/finetune" => "llama-cpp-finetune"
+    libexec.install "build/bin/main" => "llama-cpp"
```

Comment:
```suggestion
    libexec.install "build/bin/main" => "llama-cli"
```

It'd be better and more cleaner if it's `llama-cli`, `llama-server` just because it is much cleaner overall.